services/common-utils/README.md

```diff
@@ -0,0 +1,8 @@
+# Common-Utils Library
+
+This library holds low-level common utility classes that are used by iterators
+on tablet servers, microservices, and the legacy Wildfly web service.
+
+**WARNING:** Think very carefully about adding any dependencies to this
+library. Only low-level additions with little or no external dependencies
+should be added.
```

Comment:
extend this comment to include the reasoning: namely that it is expected many other services will be dependent on this one, and as such and additional dependencies or changes will potentially affect many other microservices etc.